app/src/main/java/org/sopt/hyundaicarddive/presentation/ui/home/component/CategoryBar.kt

```diff
@@ -0,0 +1,105 @@
+package org.sopt.hyundaicarddive.presentation.ui.home.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.hyundaicarddive.core.util.noRippleClickable
+import org.sopt.hyundaicarddive.presentation.type.CategoryType
+import org.sopt.hyundaicarddive.ui.theme.HYUNDAICARDDIVETheme
+import org.sopt.hyundaicarddive.ui.theme.HYUNDAICARDDIVETheme.colors
+import org.sopt.hyundaicarddive.ui.theme.HYUNDAICARDDIVETheme.typography
+
+@Composable
+fun CategoryBar(
+    modifier: Modifier = Modifier,
+    selectedIndex: Int = 0,
+    onIndexSelected: (Int) -> Unit = {}
+) {
+    val contentLists = listOf(
+        CategoryType.TOTAL.description,
+        CategoryType.DESIGN.description,
+        CategoryType.BUILDING.description,
+        CategoryType.TRIP.description,
+        CategoryType.MUSIC.description,
+        CategoryType.COOKING.description,
+        CategoryType.STYLE.description,
+        CategoryType.TECH.description,
+        CategoryType.SPECIAL.description
+    )
+
+    LazyRow(
+        modifier = modifier
+            .padding(vertical = 12.dp)
+            .fillMaxWidth(),
+        horizontalArrangement = Arrangement.spacedBy(10.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        items(contentLists.size) { index ->
+            if (index == 0) {
+                Spacer(Modifier.width(30.dp))
+            }
```

Comment:
```suggestion
        item{ Spacer(Modifier.width(30.dp) }
        
        items(contentLists.size) { index ->
            ...
        }
```

이런식으로 작성해도 괜찮을 거 같아요

Reply:
헉 더 간단한 방법 추천 너무 좋네요 ! 바로 반영할게요 ~